### DISCUSSION TITLE
How to find orders by unanswered questions?

### DISCUSSION THREAD

**Discussion by JohnWalkerx** [pretix/pretix#3089] [Q&A]
Hi, 
how can I find orders where specific questions are not answered?

What I found so far:
- I can filter for specific values of a question in the orders view.
- On text or choose-from-a-list questions I can get all orders from the question summary

**My usecase:**
I use pretix to handle registrations for a event.
For each paid order I need to do some manual tasks.
Now a would like to store this information on the task that I processed it.
To know which orders I need to process I would need to filter all orders which haven't processed already.

**Idea:**
My idea was to create a choose-from-a-list question which is only visible in the backend.
So I can easily get the orders from the question overview for a specific value.
But on a new order this question is unanswered. So I can't filter for unanswered questions.

Or I could use the comment section of a order but I also can't apply a filter "show all order which doesn't contain 'processed' in the comment"

**Questions:**
- How can I find orders with unanswered questions?
- Can I set a default value for questions?
- Is there a better way to do this?

Thanks for your input :slightly_smiling_face:

**Comment by raphaelm** [comment]
> How can I find orders with unanswered questions?

I don't think we have a way to filter for unanswered questions (other than "use an export to CSV")

> Can I set a default value for questions?

Nope. It was planned at some point and it might still happen, but there are some challenges to doing this properly with all the different data types.

> Is there a better way to do this?

Not sure, I don't think we're great at supporting custom workflows like this. Text in the "comment" field of an order will be searched when doing a search of orders, if that helps?

**Reply by JohnWalkerx** [reply]
> Text in the "comment" field of an order will be searched when doing a search of orders, if that helps?

Unfortunately this would not work because we need to find new orders which are not processed already. so the "comment" field is empty. And I think we can't search for "nothing" or "show me all orders without 'processed' in comment".

Maybe a could write a little program which uses the API and searches for orders which doesn't have the "unprocessed" or "processed" keyword in the comment and set it to "unprocessed".
Or I set the specific backend question to a default value.
